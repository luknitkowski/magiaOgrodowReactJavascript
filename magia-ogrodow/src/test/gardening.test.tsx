import renderer from 'react-test-renderer'

import Gardening from '../components/gardening';

test('Gardening renders correctly', () => {
  const gardening = renderer.create(<Gardening />).toJSON();
  expect(gardening).toMatchSnapshot();
});

