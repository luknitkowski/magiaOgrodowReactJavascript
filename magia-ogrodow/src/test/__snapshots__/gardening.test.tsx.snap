// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gardening renders correctly 1`] = `
<div>
  <div
    className="sc-dlfnbm kuQqoH"
  >
    <h1
      className="sc-hKgILt eirGSK"
    >
      
    </h1>
  </div>
  <div
    className="sc-eCssSg jmmezM"
  >
    <div
      className="sc-jSgupP czCLUL"
    >
      <p
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        “Ogród powinien docierać do głębi. Odmienić serce, zasmucić, uszlachetnić. Jego rolą jest skłonić nas do zadumy nad przemijalnością wszystkiego, co nas otacza . Ten szczególny punkt w czasie, kiedy ostatni liść już-już ma opaść, gdy ostatni płatek już-już ma się osypać, ten moment zawiera wszystko, co w życiu piękne i bolesne.”
      </p>
      <div
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <p
          className="sc-gKsewC bIHkkL signature"
        >
          Tan Twan Eng
        </p>
      </div>
    </div>
  </div>
  <div
    className="sc-iBPRYJ hRyXLA"
  >
    <div
      className="sc-bqyKva hDxFDd"
    >
      <div
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="sc-fubCfw kCEKns"
        >
          <div
            className="sc-kEjbxe kVUgzI"
          >
            <img
              className="sc-jrAGrp iDzoir"
              src="/images/project-images/first/1-min.jpg"
            />
          </div>
          <div
            className="sc-crrsfI bsqvHD"
          >
            <h2>
              Ogród przydomowy 845 m² – Rumia
            </h2>
            <h3>
              Projekt: 
              Beata Kulling- Nitkowska
            </h3>
            <p>
              Ogród został zmodernizowany ze starego ogrodu. Część roślin została zaadaptowana, jak na przykład wysoki żywopłot, dzięki temu uzyskaliśmy zieloną ścianę i intymność. Drugim zachowanym elementem są średniej wysokości żywopłoty bukszpanowe, które wyznaczyły strefy. Punktem centralnym stał się kwadrat , w którym znajduje się mini fontanna otoczona różami, szałwią i trawami. Przestrzeń ta przyciąga wzrok widza przez cały czas. Pozostałe strefy to skalniak przy altanie i trawnik na którym może odbywać się aktywność dzieci. Wprowadziliśmy rośliny kwitnące , jak : liliowce, przetaczniki, hortensje, rudbekie , rododendrony. Od wczesnej wiosny do jesieni cały czas coś kwitnie.
            </p>
            <button
              className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-label"
              >
                SPRAWDŻ PROJEKT
              </span>
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      className="sc-bqyKva hDxFDd"
    >
      <div
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="sc-fubCfw kCEKns"
        >
          <div
            className="sc-crrsfI bsqvHD"
          >
            <h2>
              Ogród przydomowy
            </h2>
            <h3>
              Projekt: 
              Beata Kulling- Nitkowska
            </h3>
            <p>
              Przekształcenie zaniedbanego , starego ogrodu polegało na uporządkowaniu przestrzeni. Wprowadzono linie proste i geometryczne donice, dzięki temu pozbyliśmy się chaosu i zastanego bałaganu. Jak to w starszych ogrodach pozostawiliśmy roślinność , którą można poprzez odpowiednią pielęgnację przywrócić do życia . Takim elementem był na pewno wysoki żywopłot. Na życzenie klienta zaproponowane rośliny na rabaty wymagają minimum pracy.
            </p>
            <button
              className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-label"
              >
                SPRAWDŻ PROJEKT
              </span>
            </button>
          </div>
          <div
            className="sc-kEjbxe kVUgzI"
          >
            <img
              className="sc-jrAGrp iDzoir"
              src="/images/project-images/second/1-min.jpg"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="sc-bqyKva hDxFDd"
    >
      <div
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="sc-fubCfw kCEKns"
        >
          <div
            className="sc-kEjbxe kVUgzI"
          >
            <img
              className="sc-jrAGrp iDzoir"
              src="/images/project-images/third/1-min.jpg"
            />
          </div>
          <div
            className="sc-crrsfI bsqvHD"
          >
            <h2>
              Ogród przydomowy - Mrzezino,
            </h2>
            <h3>
              Projekt: 
              Zofia Dzielińska
            </h3>
            <p>
              W tym ogrodzie byliśmy wykonawcami zaplanowanego już ogrodu przez naszą koleżankę , z którą współpracujemy Zofią Dzielińską. Zaproponowane owalne kształty , przenikające i stykające się z sobą dały spójność przestrzeni. Ogród kwitnący, tętniący kolorem i życiem wielu owadów. W tym ogrodzie nie posadziliśmy , ani jednego iglaka . Oprócz mnóstwa bylin, znalazł swoje miejsce warzywniak i szklarenka.
            </p>
            <button
              className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-label"
              >
                SPRAWDŻ PROJEKT
              </span>
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      className="sc-bqyKva hDxFDd"
    >
      <div
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="sc-fubCfw kCEKns"
        >
          <div
            className="sc-crrsfI bsqvHD"
          >
            <h2>
              Ogród nad jeziorem - Nadole
            </h2>
            <h3>
              Projekt: 
              Zofia Dzielińska
            </h3>
            <p>
              Pierwszy nasz tak duży ogród, dlatego na zawsze pozostanie w mojej pamięci. Przedstawiana działka ma kształt wąskiego prostokąta, z lekkim spadkiem w kierunku jeziora. Zastaliśmy już położone nawierzchnie . Musieliśmy wkomponować się w istniejące wygrodzone przestrzenie. Ogród podzieliliśmy na trzy strefy . Pierwsza to przedogródek, reprezentacyjny zapraszający do domu . Zaproponowana roślinność biegnie w kształcie fali wychodzącej z pierwszego podjazdu , biegnącej wzdłuż dwóch wejść do budynku. Znalazły tu miejsce róże okrywowe, berberysy , tuje kuliste. Druga strefa znajduje się z boku budynku i ma charakter leśny , gdyż została wkomponowana w już istniejące sosny. Znalazły tam miejsce rośliny tolerujące cień : Rododendrony, żurawki, hosty, rodgersje. Trzecia wydzielona przestrzeń znajduje się po drugiej stronie budynku, blisko jeziora. W tej części ogrodu przez spadek terenu stworzył się taras , który kończy mała skarpa Zagospodarowana ona została roślinami zadarniającymi : jałowce, skalniaki, runianka , żurawki.
            </p>
            <button
              className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-label"
              >
                SPRAWDŻ PROJEKT
              </span>
            </button>
          </div>
          <div
            className="sc-kEjbxe kVUgzI"
          >
            <img
              className="sc-jrAGrp iDzoir"
              src="/images/project-images/fourth/1-min.jpg"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="sc-bqyKva hDxFDd"
    >
      <div
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="sc-fubCfw kCEKns"
        >
          <div
            className="sc-kEjbxe kVUgzI"
          >
            <img
              className="sc-jrAGrp iDzoir"
              src="/images/project-images/fifth/1-min.jpg"
            />
          </div>
          <div
            className="sc-crrsfI bsqvHD"
          >
            <h2>
              Przedogródek - Rumia
            </h2>
            <h3>
              Projekt: 
              Zofia Dzielińska
            </h3>
            <p>
              Mały przedogródek w kształcie litery L. Mały , ale dużo się się dzieje. Rabty wydzieliły łuki i dla ich podkreślenia zastosowano dwa rodzaje kamienia, ciemny i jasny. Jedna część skąpana w słońcu , dlatego miejsce znalazły tam trawy i w różnych okresach kwitnące byliny. Z drugiej strony budynku poprzez zastane drzewa i krzewy zamieszkały rośliny cienia i półcienia. Zakątek ten mieni się kolorami . Dwie części przedogródka połączył trawnik w kształcie księżyca.
            </p>
            <button
              className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-label"
              >
                SPRAWDŻ PROJEKT
              </span>
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      className="sc-bqyKva hDxFDd"
    >
      <div
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="sc-fubCfw kCEKns"
        >
          <div
            className="sc-crrsfI bsqvHD"
          >
            <h2>
              Przestrzeń publiczna przed blokiem - Rumia
            </h2>
            <h3>
              Projekt: 
              Zofia Dzielińska
            </h3>
            <p>
              Czasami trzeba zaprojektować przestrzeń publiczną , która należy do Wszystkich. Takie miejsca oprócz funkcji dekoracyjnej muszą być funkcjonalne. Zadbane wejścia , czy przdogródki w blokach podnoszą atrakcyjność takiego miejsca i cieszą samych mieszkańców, wpływając pozytywnie na ich samopoczucie , klimat oraz różnorodność biologiczną. Proponowane rośliny powinny być samoobsługowe , jak i odporne na warunki pogodowe np. susze.
            </p>
            <button
              className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-label"
              >
                SPRAWDŻ PROJEKT
              </span>
            </button>
          </div>
          <div
            className="sc-kEjbxe kVUgzI"
          >
            <img
              className="sc-jrAGrp iDzoir"
              src="/images/project-images/sixth/1-min.jpg"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="sc-dlfnbm gaehrs"
  >
    <h1
      className="sc-hKgILt eirGSK"
    >
      
    </h1>
  </div>
</div>
`;
